climada/hazard/trop_cyclone.py

```diff
@@ -646,37 +646,37 @@ def compute_windfields(track, centroids, model, metric="equirect"):
     t_rad[:] = estimate_rmw(t_rad, t_cen) * NM_TO_KM
 
     # translational speed of track at every node
-    v_trans = _vtrans(t_lat, t_lon, t_tstep, metric=metric)
-    v_trans_norm = v_trans[0]
+    [v_trans_norm, v_trans] = _vtrans(t_lat, t_lon, t_tstep, metric=metric)
 
     # adjust pressure at previous track point
     prev_pres = t_cen[:-1].copy()
     msk = (prev_pres < 850)
     prev_pres[msk] = t_cen[1:][msk]
 
     # compute b-value and derive (absolute) angular velocity
+    v_ang_norm = np.zeros((npositions, nreachable), dtype=np.float64)
```

Comment:
This does not affect the performance, but only avoids arrays with `npositions - 1` entries. It also improves the clarity of the code since the array is initialized before the `if ... else ...` block below.